tasm-tour/src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -1,25 +1,29 @@
 import Button from './Button';
 import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/24/outline';
 
-export default function ButtonPanel({ exhibitID }) {
+export default function ButtonPanel({ setExhibitID, nextExhibit, prevExhibit }) {
   return (
     <div className="flex justify-around p-4">
-      <Button
-        label="Prev"
-        onClick={() => console.log('Previous clicked')}
-        icon={ChevronLeftIcon}
-        iconProps={{ className: "w-7 h-7" }}
-        iconPosition="left"
-        className="btn rounded-l-full pl-1 pr-3 py-1 text-xl drop-shadow-[-2px_3px_4px_rgba(0,0,0,0.25)]"
-      />
-      <Button
-        label="Next"
-        onClick={() => console.log('Next clicked')}
-        icon={ChevronRightIcon}
-        iconProps={{ className: "w-7 h-7" }}
-        iconPosition="right"
-        className="btn rounded-r-full pr-1 pl-3 py-1 text-xl drop-shadow-[2px_3px_4px_rgba(0,0,0,0.25)]"
-      />
+      {prevExhibit ?
+        <Button
+          label="Prev"
+          onClick={() => setExhibitID(prevExhibit)}
+          icon={ChevronLeftIcon}
+          iconProps={{ className: "w-7 h-7" }}
+          iconPosition="left"
+          className="btn rounded-l-full pl-1 pr-3 py-1 text-xl drop-shadow-[-2px_3px_4px_rgba(0,0,0,0.25)]"
+        />
+        : null}
+      {nextExhibit ?
```

Comment:
is this conditional readable? Should we have a default 'next'/'prev' to use rather than hiding the buttons?

Reply:
It looks pretty straight-forward to me! Just checking for a "next" or "prev" exhibit, then rendering the button if one is present/hiding the button if not.